### PR TITLE
NN-2186 add recurring info to the upload page

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: backend/controllers/addAppointmentDetails.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const moment = require('moment')
 const config = require('../config')
 const { serviceUnavailableMessage } = require('../common-messages')
 const { DATE_TIME_FORMAT_SPEC, DAY_MONTH_YEAR, buildDateTime } = require('../../src/dateHelpers')
-
+const { properCaseName } = require('../utils')
```
Comment (jsrobertson) [inline backend/controllers/addAppointmentDetails.js]: Not a show stopper but wondering if this is better off being called `capitalise` now it's being used for things other than names.

Comment (JamieDraperUK) [reply backend/controllers/addAppointmentDetails.js]: That's a good spot!

Comment (JamieDraperUK) [reply backend/controllers/addAppointmentDetails.js]: Will change it on the next PR

Comment (jsrobertson) [reply backend/controllers/addAppointmentDetails.js]: Oh, there is already a `capitalize` in utils

**Pull request merged**